### PR TITLE
Update tests for "Get a quote" button in the header

### PR DESCRIPTION
**Pull request by SofiiaRostkovych**

### CODE REVIEW

File: ui_tests_playwright/automation/test/ui/generalComponents/header/headerMainSite/HeaderElementsColorChecks.test.ts [hunk 1]
```diff
@@ -266,8 +266,13 @@ test(`Check the header information from the "Header" container on all pages @des
...
 		await expect(contactsButton).toHaveText('Contacts');
-
-		await expect(getAQuoteButton).toHaveText('Get a quote');
+        
+		if (webflowPages.includes(url)){
+			await expect(getAQuoteButton).toHaveText('Get a quote');
+			}else{
+				await expect(getAQuoteButton).toHaveText('Book a strategy call');
+			};
+
```
Comment (Valerii-Kryvokhyzha) [inline ui_tests_playwright/automation/test/ui/generalComponents/header/headerMainSite/HeaderElementsColorChecks.test.ts:275]: We tack this btn text based on BW/Webflow ? Not by 50% random chance of displaying any version of btn to user?

**Review by Valerii-Kryvokhyzha — CHANGES_REQUESTED** [review]

File: ui_tests_playwright/automation/test/ui/generalComponents/header/headerMainSite/HeaderElementsColorChecks.test.ts (hunk 1 above)
Comment (SofiiaRostkovych) [reply ui_tests_playwright/automation/test/ui/generalComponents/header/headerMainSite/HeaderElementsColorChecks.test.ts:275]: on BW 'Book a strategy call' is the only version of btn displayed
on webflow pages btn was not updated, so it's 'Get a quote'

**Review by Valerii-Kryvokhyzha — APPROVED** [review]

File: ui_tests_playwright/automation/test/ui/generalComponents/header/headerMainSite/HeaderElementsColorChecks.test.ts [hunk 2]
```diff
@@ -267,7 +273,11 @@ test(`Check the header information from the "Header" container on all pages @des
 		await expect(pricingButton).toHaveText('Pricing');
 		await expect(contactsButton).toHaveText('Contacts');
 
-		await expect(getAQuoteButton).toHaveText('Get a quote');
+		if (webflowPages.includes(url)) {
```
Comment (DaniilVostrikov) [inline ui_tests_playwright/automation/test/ui/generalComponents/header/headerMainSite/HeaderElementsColorChecks.test.ts:276]: please, try to use ternary expression here

File: ui_tests_playwright/automation/test/ui/generalComponents/header/headerMainSite/HeaderFunctionalChecks.test.ts [hunk 3]
```diff
@@ -227,7 +228,11 @@ test(
 			await baseDriverSteps.goToUrl(url);
 			await headerMenuSteps.clickOnBurgerMenu();
 			await header.getByTestId(MainSiteButtons.GetAQuote).click();
-			await baseDriverSteps.checkUrl(UrlProvider.urlBuilder(UrlPath.GetAQuote));
+			if (webflowPages.includes(url)) {
+				await baseDriverSteps.checkUrl(UrlProvider.urlBuilder(UrlPath.GetAQuote));
```
Comment (DaniilVostrikov) [inline ui_tests_playwright/automation/test/ui/generalComponents/header/headerMainSite/HeaderFunctionalChecks.test.ts:232]: same here

**Review by DaniilVostrikov — CHANGES_REQUESTED** [review]

**Review by DaniilVostrikov — APPROVED** [review]

**Pull request merged**